libcontainerd/client_windows.go

```diff
@@ -219,6 +220,21 @@ func (clnt *client) Create(containerID string, checkpoint string, checkpointDir
 			if strings.ToLower(o) == "ro" {
 				mds[i].ReadOnly = true
 			}
+			optionParts := strings.SplitN(o, "=", 2)
+			switch optionParts[0] {
+			case "maximum_bandwidth":
+				bandwidth, err := strconv.Atoi(optionParts[1])
+				if err != nil {
+					return err
```

Comment:
Might be more useful to give some context as to it's failed on maximum_bandwidth.